### ISSUE TITLE
Fails with mocha dependencies

### ISSUE THREAD

**Issue by amilajack** [fengb/require-optional#2]
When installing this npm package into a project that already has mocha, sinon-chai, jade, and other dependencies, it causes the build to fail

**Comment by amilajack** [comment]
Log:

```
] ERROR in ./~/require-optional/test/index.js
[0] Module not found: Error: Cannot resolve module 'sinon-chai' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/require-optional/test
[0]  @ ./~/require-optional/test/index.js 2:18-39
[0]
[0] ERROR in ./~/mocha/index.js
[0] Module not found: Error: Cannot resolve 'file' or 'directory' ./lib-cov/mocha in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/mocha
[0]  @ ./~/mocha/index.js 2:4-30
[0]
[0] ERROR in ./~/jade/index.js
[0] Module not found: Error: Cannot resolve 'file' or 'directory' ./lib-cov/jade in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/jade
[0]  @ ./~/jade/index.js 3:4-29
[0]
[0] ERROR in ./~/jade/lib/filters.js
[0] Module not found: Error: Cannot resolve module 'sass' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/jade/lib
[0]  @ ./~/jade/lib/filters.js 24:15-30
[0]
[0] ERROR in ./~/jade/lib/filters.js
[0] Module not found: Error: Cannot resolve module 'stylus' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/jade/lib
[0]  @ ./~/jade/lib/filters.js 35:17-34
[0]
[0] ERROR in ./~/jade/lib/filters.js
[0] Module not found: Error: Cannot resolve module 'less' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/jade/lib
[0]  @ ./~/jade/lib/filters.js 50:4-19
[0]
[0] ERROR in ./~/jade/lib/filters.js
[0] Module not found: Error: Cannot resolve module 'coffee-script' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/jade/lib
[0]  @ ./~/jade/lib/filters.js 94:13-37
[0]
[0] ERROR in ./~/mocha/~/minimatch/test/basic.js
[0] Module not found: Error: Cannot resolve module 'tap' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/mocha/node_modules/minimatch/test
[0]  @ ./~/mocha/~/minimatch/test/basic.js 6:10-24
[0]
[0] ERROR in ./~/mocha/~/minimatch/test/brace-expand.js
[0] Module not found: Error: Cannot resolve module 'tap' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/mocha/node_modules/minimatch/test
[0]  @ ./~/mocha/~/minimatch/test/brace-expand.js 1:10-24
[0]
[0] ERROR in ./~/mocha/~/minimatch/test/caching.js
[0] Module not found: Error: Cannot resolve module 'tap' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/mocha/node_modules/minimatch/test
[0]  @ ./~/mocha/~/minimatch/test/caching.js 2:10-24
[0]
[0] ERROR in ./~/mocha/~/minimatch/test/defaults.js
[0] Module not found: Error: Cannot resolve module 'tap' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/mocha/node_modules/minimatch/test
[0]  @ ./~/mocha/~/minimatch/test/defaults.js 6:10-24
[0]
[0] ERROR in ./~/mocha/~/minimatch/test/extglob-ending-with-state-char.js
[0] Module not found: Error: Cannot resolve module 'tap' in /Users/amila/Documents/Projects/popcorn-desktop-experimental/node_modules/mocha/node_modules/minimatch/test
[0]  @ ./~/mocha/~/minimatch/test/extglob-ending-with-state-char.js 1:11-25
[0] webpack: bundle is now VALID.
```

**Comment by amilajack** [comment]
Found a solution:

I deleted tests folder (`node_modules/require-optional/test`) and it worked properly without errors. I'll make a PR for this.

**Comment by fengb** [comment]
Fixed via #3
